src/renderer/components/dock/terminal.store.ts

```diff
@@ -117,15 +117,18 @@ export class TerminalStore extends Singleton {
 
       await when(() => this.connections.has(tab.id));
 
-      const rcIsFinished = when(() => this.connections.get(tab.id).shellRunCommandsFinished);
+      const shellIsReady = when(() => this.connections.get(tab.id).isReady);
       const notifyVeryLong = setTimeout(() => {
-        rcIsFinished.cancel();
-        Notifications.info("Terminal shell is taking a long time to complete startup. Please check your .rc file. Bypassing shell completion check.", {
-          timeout: 4_000,
-        });
+        shellIsReady.cancel();
+        Notifications.info(
+          "If terminal shell is not ready please check your shell init files, if applicable. You may see this message if you have set a custom shell prompt.",
```

Comment:
custom shell prompt is no longer relevant?

Reply:
yeah fair enough, it might still be relevant but probably not.